src/site/_includes/components/Hero.js

```diff
@@ -13,26 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-const {html} = require('common-tags');
-const getSrcsetRange = require('../../_utils/get-srcset-range');
+const {Img} = require('./Img');
 
 module.exports = ({hero, alt, heroPosition, heroFit = 'cover'}) => {
-  const srcsetRange = getSrcsetRange();
+  const className = `w-hero w-hero--${heroFit}${
+    heroPosition ? ` w-hero--${heroPosition}` : ''
+  }`;
 
-  // prettier-ignore
-  return html`
-    <img
-      class="w-hero w-hero--${heroFit} ${heroPosition ? `w-hero--${heroPosition}` : ''}"
-      width="1600"
-      height="480"
-      sizes="100vw"
-      srcset="${srcsetRange.map((width) => html`
-        ${hero}?auto=format&fit=max&w=${width} ${width}w,
-      `)}"
-      src="${hero}"
-      alt="${alt}"
-      loading="lazy"
-    />
-  `;
+  return Img({
+    src: hero,
+    alt: typeof alt === 'string' ? alt : 'Hero Image',
```

Comment:
Even if empty string?

Reply:
This only exists because some post(s) send an object that breaks the type check (i cant pin down the file cause i dont have access to the page object). Separately the short code requires an alt tag, so this was a decent one.

Reply:
@MichaelSolati if you change the shortcode to use `function() {}` instead of arrow functions then you can inspect `this.page` inside of it to see which page is breaking.